SCClassLibrary/Common/Core/Kernel.sc

```diff
@@ -700,8 +746,8 @@ Interpreter {
 		protect {
```

Comment:
could you undo the formatting here? I think it helps for future readability to know – at least – which files were actually touched an which weren't.